### PR TITLE
[ci] link to batch from job page

### PR DESCRIPTION
**Pull request by daniel-goldstein**

### CODE REVIEW

**Comment by daniel-goldstein** [comment]
bump

**Comment by jigold** [comment]
I think this is a temporary fix as we want Batch to directly return these pages. @CDiaz96 and I will start thinking about how this should be done in Batch more generally tomorrow.

**Comment by daniel-goldstein** [comment]
Ok, happy to close if you're working on a solution.

**Comment by jigold** [comment]
I think your solution is better right now.

File: ci/ci/templates/job.html [hunk 1]
```diff
@@ -3,6 +3,9 @@
 {% block content %}
     <h1>Batch {{ batch_id }} Job {{ job_id }}</h1>
 
+    <form method="GET" action="{{ base_path }}/batches/{{ batch_id }}">
+      <button>Back to batch</button>
+    </form>
```
Comment (jigold) [inline ci/ci/templates/job.html:8]: Sorry to nitpick. Can we get another new line after the form?

**Review by jigold — CHANGES_REQUESTED** [review]

**Review by jigold — APPROVED** [review]

**Pull request merged**